speedify.py

```diff
@@ -269,7 +302,89 @@ def logout():
     return find_state_for_string(jret["state"])
 
 
-### Getter functions ###
+def privacy(category: str, is_on: bool):
```

Comment:
I think you can remove the privacy function. We just remove the DoH option from the Speedify UI in the last version, so I don't think it needs to be here either. Also this privacy function doesn't follow the existing pattern of having a function for each setting. All the other privacy settings, such as encryption or killswitch have their own dedicated function.